src/main.rs

```diff
@@ -481,9 +481,8 @@ fn elf_to_tbf<W: Write>(
             && input.ehdr.entry < (section.shdr.addr + section.shdr.size)
             && (section.shdr.name.find("debug")).is_none()
         {
-            // panic in case we detect entry point in multiple sections.
             if entry_point_found {
-                panic!("Duplicate entry point in {} section", section.shdr.name);
+                println!("Duplicate entry point in {} section", section.shdr.name);
```

Comment:
This behavior should probably be gated on a command-line flag.  For non-otbn apps, this is definately an error.

One could also question whether or not an OTBN app should be packaged directly in a tock app like this or whether an OTBN app should be a data section inside of a "proper" tock app and be given to the OTBN driver by means of a syscall.

I would argue the former is acceptable for hacky development, but the latter should actually be the long-term goal.  An OTBN program _is not_ a tock app and cannot ever execute under tock.  It is, at best, a data payload needed by some other tock app.